src/lib/logical_query_plan/jit_aware_lqp_translator.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "lqp_translator.hpp"
+#include "operators/jit_operator/operators/jit_compute.hpp"
+#include "operators/jit_operator_wrapper.hpp"
+
+namespace opossum {
+
+/* This class can be used as a drop-in replacement for the LQPTranslator.
```

Comment:
I would rather call it "specialization" instead of "replacement", because it inherits from LQPTranslator